tests/xrt/soft_kernel_hello/Makefile

```diff
@@ -0,0 +1,4 @@
+
+RUN_SHELL := $(shell ./runit.sh)
+
+$(info $(RUN_SHELL))
```

Comment:
Please do not call scripts from Makefile. 
Use xclbin.mk for compiling PL kernels with v++ and CMakeLists.txt for host code. See examples like:
XRT/tests/xrt/13_add_one
XRT/tests/xrt/100_ert_ncu

Reply:
done